src/lambdas/initiateAthenaQuery/createQuerySql.ts

```diff
@@ -178,23 +170,13 @@ const formatIdWhereStatement = (identifierType: IdentifierTypes) => {
 
 const generateQueryParameters = (
   identifiers: string[],
-  dateFrom: string,
-  dateTo: string
+  dates: string[]
 ): string[] => {
   const queryParameters = identifiers.map((identifier) => `'${identifier}'`)
-  queryParameters.push(formatDateFrom(dateFrom))
-  queryParameters.push(formatDateTo(dateTo))
+  dates.forEach((date) => queryParameters.push(`'${formatDateForQuery(date)}'`))
   return queryParameters
 }
 
-const formatDateFrom = (dateFrom: string): string => {
-  const splitDateFrom = dateFrom.split('-')
-  splitDateFrom.push('00')
-  return `'${splitDateFrom.join('/')}'`
-}
-
-const formatDateTo = (dateTo: string): string => {
-  const splitDateTo = dateTo.split('-')
-  splitDateTo.push('23')
-  return `'${splitDateTo.join('/')}'`
+const formatDateForQuery = (dateFrom: string): string => {
+  return dateFrom.replaceAll('-', '/')
```

Comment:
Yes, talking to Tom, we realised we were unnecessarily partitioning for hours, only to then say "tell me everything on this day".

You'll notice a change to the Athena table as part of this PR, where we switch to partitioning by day instead of hour.

If at some point in the future we get the analysts to tell us a particular time of day when we're looking for an event, we can put this partitioning back, we can put it back, but for now it just wasn't adding anything, and would have made the query unnecessarily complicated when we added multiple dates.